code/modules/mob/living/basic/space_fauna/bear/_bear.dm

```diff
@@ -70,6 +70,10 @@
 	honey_food.forceMove(get_turf(src))
 
 //SPACE BEARS! SQUEEEEEEEE~     OW! FUCK! IT BIT MY HAND OFF!!
+/mob/living/basic/bear/bitrunning
+	faction = list(ROLE_SYNDICATE)
+	ai_controller = /datum/ai_controller/basic_controller/trooper
```

Comment:
Why the change of AI controller? It's not explained in the PR body.

Reply:
is only the faction flag required? I'll test that.

Reply:
Why the syndicate domain? Did hostile not work?

Reply:
> Why the syndicate domain? Did hostile not work?

they were attacking `GLITCH` antagonists so I just copied over the behavior from other domains

Reply:
glitch antags (assuming you mean cybercops etc?) should have the npc friendly component

Reply:
> glitch antags (assuming you mean cybercops etc?) should have the npc friendly component

I tested it, before my changes `GLITCH` antags (yes, cybercops, cybertac and net guardian prime) were being attacked by the bears